kafka/channel/pkg/dispatcher/dispatcher.go

```diff
@@ -90,7 +90,7 @@ func NewDispatcher(ctx context.Context, args *KafkaDispatcherArgs) (*KafkaDispat
 
 	producer, err := sarama.NewAsyncProducer(args.Brokers, conf)
 	if err != nil {
-		return nil, fmt.Errorf("unable to create kafka producer: %v", err)
+		return nil, fmt.Errorf("unable to create kafka producer for brokers %v : %v", args.Brokers, err)
```

Comment:
Minor irrelevant improvement that I noticed while testing against namespace scoped channels. Wanted to see what broker is used. Relevant: https://github.com/knative/eventing-contrib/issues/1189

Reply:
Kafka broker?  

Broker is an overloaded term

Reply:
Ok, let me change to "Kafka bootstrap servers"

Reply:
Done